charts/argo-rollouts/templates/dashboard/clusterrole.yaml

```diff
@@ -40,6 +40,17 @@ rules:
       - get
       - list
       - watch
+  # deployments and podtemplates read access needed for workload reference support
+  - apiGroups:
+      - ""
```

Comment:
Is this intentional?

Reply:
I was copying from here as @huikang recommended: https://github.com/argoproj/argo-rollouts/blob/55a041a6b3d088cf9ddcfc4121ad8c97880489db/manifests/role/argo-rollouts-clusterrole.yaml#L60-L69

Reply:
I think it's because `PodTemplate` doesn't have an API group?

Reply:
Ah, I see :+1: